processing/src/main/java/org/apache/druid/segment/serde/ColumnPartSerde.java

```diff
@@ -39,7 +39,8 @@
     @JsonSubTypes.Type(name = "floatV2", value = FloatNumericColumnPartSerdeV2.class),
     @JsonSubTypes.Type(name = "longV2", value = LongNumericColumnPartSerdeV2.class),
     @JsonSubTypes.Type(name = "doubleV2", value = DoubleNumericColumnPartSerdeV2.class),
-    @JsonSubTypes.Type(name = "null", value = NullColumnPartSerde.class)
+    @JsonSubTypes.Type(name = "null", value = NullColumnPartSerde.class),
+    @JsonSubTypes.Type(name = "standard", value = StandardTypeColumnPartSerde.class)
```

Comment:
What're the semantics of the standard column part serde trying to do?  If it's standard, does it subsume all of the others?